src/Scenes/Opening.cpp

```diff
@@ -301,7 +312,18 @@ std::uint16_t Opening::poll(const sf::Event& e)
 
             case sf::Keyboard::Num0:
             {
+                _debugEnabled = !_debugEnabled;
```

Comment:
Doh! I think I looked over this. I implemented the same thing for the 'H' key. I'll incorporate my `H` key functionality into this once this PR is merged.